library/src/scala/quoted/Quotes.scala

```diff
@@ -3816,7 +3816,7 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
       *
       *  @param parent The owner of the binding
       *  @param name The name of the binding
-      *  @param flags extra flags to with which the symbol should be constructed
+      *  @param flags extra flags to with which the symbol should be constructed. `Case` flag will be added. Can be `Case`
```

Comment:
If Case is always added and Case is the only valid flag, it sounds like this parameter is not actually necessary?

Reply:
In the future, we might have more flags (see `validBindFlags`). 

We could add an overload of `newBind` that does not require the flags.